spec/features/burgers_features/burger_add_spec.rb

```diff
@@ -1,17 +1,33 @@
 require "rails_helper"
 
 feature "User visits new burger page" do
+  let(:restaurant) { Restaurant.create(name: "restaurant", location: "Boston", dining_type: "Might be a selction", description: "Really long lines and really long text", hours: "10am-10pm") }
```

Comment:
You need to update your hound file for line length. This would still be innapropriate, and this restaurant should span multiple lines rather than 1